pkg/controller/scylladbmanagerclusterregistration/controller.go

```diff
@@ -332,6 +365,24 @@ func (smcrc *Controller) enqueueThroughScyllaDBDatacenter(depth int, obj kubeint
 	smcrc.handlers.Enqueue(depth+1, smcr, op)
 }
 
+func (smcrc *Controller) enqueueThroughScyllaDBCluster(depth int, obj kubeinterfaces.ObjectInterface, op controllerhelpers.HandlerOperationType) {
+	sc := obj.(*scyllav1alpha1.ScyllaDBCluster)
+
+	smcrName, err := naming.ScyllaDBManagerClusterRegistrationNameForScyllaDBCluster(sc)
+	if err != nil {
+		apimachineryutilruntime.HandleError(err)
+		return
+	}
+
+	smcr, err := smcrc.scyllaDBManagerClusterRegistrationLister.ScyllaDBManagerClusterRegistrations(sc.Namespace).Get(smcrName)
+	if err != nil {
+		return
+	}
```

Comment:
Shouldn't we ignore the error only if the CR is not found? In case of some other, transient error here, we might ignore that and miss the enqueue. Perhaps calling `apimachineryutilruntime.HandleError(err)` when it's not `NotFound` could improve visibility at least?

Reply:
done